src/Kentico.Xperience.Lucene/Admin/Client/src/templates/LuceneIndexConfigurationFormComponent.tsx

```diff
@@ -1,309 +1,430 @@
 import { type FormComponentProps } from '@kentico/xperience-admin-base';
 import {
-  type ActionCell,
-  Button,
-  ButtonType,
-  CellType,
-  ColumnContentType,
-  Input,
-  Stack,
-  type StringCell,
-  Table,
-  type TableAction,
-  type TableCell,
-  type TableColumn,
-  type TableRow,
-  TextArea,
+    type ActionCell,
+    Button,
+    ButtonType,
+    CellType,
+    ColumnContentType,
+    Input,
+    Stack,
+    type StringCell,
+    Table,
+    type TableAction,
+    type TableCell,
+    type TableColumn,
+    type TableRow,
 } from '@kentico/xperience-admin-components';
-import React, { useEffect, useState } from 'react';
+import React, { type CSSProperties, useEffect, useState } from 'react';
+import { IoCheckmarkSharp } from 'react-icons/io5';
+import { MdOutlineCancel } from 'react-icons/md';
+import { RxCross1 } from 'react-icons/rx';
+import Select, { type ActionMeta, type ClearIndicatorProps, components, type CSSObjectWithLabel, type GroupBase, type MultiValue, type MultiValueRemoveProps, type OptionProps, type StylesConfig } from 'react-select'; // eslint-disable-line sort-imports
```

Comment:
We don't need to disable the eslint rule here, just fix the order of imported types.